src/Page/CIFpage/index.js

```diff
@@ -0,0 +1,246 @@
+              </li>
+            </ul>
+          </div>
+          <div className="sub-section">
+            <span className="mb-3 fw-bold d-block">
+              Academic administration
+            </span>
+            <ul>
+              <li>
+                Undertaking various administrative responsibilities as required,
+                including assessment of progress, student attendance, teaching
+                quality, student experience, etc
+              </li>
+              <li>
+                Participating in regular interactions with other CIF fellows and
+                be a part of the peer-to-peer learning process
+              </li>
+            </ul>
+          </div>
+          <div className="sub-section mb-0">
+            <span className="mb-3 fw-bold d-block">
+              Documentation and reporting
+            </span>
+            To document your journey as a fellow and prepare a report at the end
+            of the Fellowship
+          </div>
+        </div>
+      </div>
+      <div className="application text-center d-flex flex-column align-items-center justify-content-center  mx-3 mx-md-0">
+        <h3 className="mb-3">Application Process</h3>
+        <hr className="heading-hr mb-3" />
+        <div className="container text-center w-50 mt-3">
+          <div className="row my-2 d-flex flex-column flex-md-row align-items-center justify-content-around">
+            <div className="d-flex fw-600 align-items-center mb-3  justify-content-center">
+              <img src={form} alt="" className="mr-0.5" />
+              Fill the Application form
+            </div>
+            <div className="sign position-relative mb-3  justify-content-center">
+              <img src={greaterThan} alt="" className="mr-0.5" />
+            </div>
+            <div className="d-flex fw-600 align-items-center mb-3  justify-content-center">
+              <img src={code} alt="" className="mr-0.5" />
+              Basic coding test
+            </div>
+            <div className="sign position-relative mb-3  justify-content-center">
+              <img src={greaterThan} alt="" className="mr-0.5" />
+            </div>
+            <div className="d-flex fw-600 align-items-center mb-3  justify-content-center">
+              <img src={chat} alt="" className="mr-0.5" />
+              In-person/Online conversation
+            </div>
+          </div>
+        </div>
+        <span className="w-40 mb-3 section-para">
+          Are you ready to nurture programming skills in the next generation of
+          India’s youth with your expertise
+        </span>
+        <button
+          className="btn btn-primary text-white mb-1 py-2 px-8 regular-btn fw-bold"
+          onClick={(e) => {
+            e.preventDefault();
+            window.open(
+              "https://docs.google.com/forms/d/e/1FAIpQLSfPvhoUHeHBl_UxZhn2kVurMNhdfYvG8KBUdl3GyXEvCEYh5A/viewform",
+              "_blank"
+            );
```

Comment:
This `onClick` handler is being used quite often across files.  I recommend defining some named function that you could refer to in case you decide to handle this differently in the future (e.g., open in a modal instead of a new tab/window).